Add a reference to cupla to readme (#914)
alpaka-group/alpaka: README.md

The whole alpaka library can be included with: `#include <alpaka/alpaka.hpp>`
Code that is not intended to be utilized by the user is hidden in the `detail` namespace.

> Another library [*cupla*](https://github.com/ComputationalRadiationPhysics/cupla) provides a very CUDA-like user interface to alpaka.

Reviewer: I would write this more personal, since we write cupla:
```suggestion
Furthermore, for a CUDA-like experience when adopting Alpaka we provide the library [*cupla*](https://github.com/ComputationalRadiationPhysics/cupla).
```

Author: I feel this is indeed better, changed and rebased. Thanks!

> It enables a simple and straightforward way of porting existing CUDA applications to alpaka and thus to a variety accelerators.

Reviewer 2: a variety *of* accelerators.

Author: Thanks, fixed.
